### PR TITLE
indexing: update libraries mapping

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Adds analyzer on name field.
* Improves global analyzer.

How to test:

* `script/setup`
* http://localhost:5000/admin/records/libraries
* Find a term and check if the result is correct (value in fields name or description)

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
Co-Authored-by: Peter Weber <peter.weber@rero.ch>

### CODE REVIEW

File: tests/data/data.json [hunk 1]
```diff
@@ -415,6 +415,107 @@
       }
     ]
   },
+  "lib5": {
+    "$schema": "https://ils.rero.ch/schema/libraries/library-v0.0.1.json",
+    "address": "Centre culturel",
+    "code": "AOSTE",
+    "email": "reroilstest+aoste@gmail.com",
+    "name": "Bibliothèque d'Aoste",
```
Comment (BadrAly) [inline tests/data/data.json]: The `org2` is the `district of Sion`, better to change the name of the library from `Aoste` to one of these:
`Arbaz`, `Grmisuat`, `Savièse`, ou `Vesyonnaz`

File: tests/fixtures/organisations.py [hunk 2]
```diff
@@ -160,6 +160,24 @@ def lib_sion(app, org_sion, lib_sion_data):
     return lib
 
 
+@pytest.fixture(scope="module")
+def lib_aoste_data(data):
```
Comment (BadrAly) [inline tests/fixtures/organisations.py]: same here

File: tests/fixtures/organisations.py [hunk 3]
```diff
@@ -160,6 +160,24 @@ def lib_sion(app, org_sion, lib_sion_data):
...
+    """Aoste library data."""
+    return deepcopy(data.get('lib5'))
+
+
+@pytest.fixture(scope="module")
+def lib_aoste(app, org_sion, lib_aoste_data):
```
Comment (BadrAly) [inline tests/fixtures/organisations.py]: same here

File: tests/data/data.json (hunk 1 above)
Comment (Garfield-fr) [reply tests/data/data.json]: done

File: tests/fixtures/organisations.py (hunk 2 above)
Comment (Garfield-fr) [reply tests/fixtures/organisations.py]: done

File: tests/fixtures/organisations.py (hunk 3 above)
Comment (Garfield-fr) [reply tests/fixtures/organisations.py]: done

**Pull request merged**